packages/story-editor/src/utils/storyPageToCanvas.js

```diff
@@ -43,7 +43,7 @@ async function storyPageToCanvas(page, { width = 400, ...options }) {
   const canvas = await htmlToImage.toCanvas(node, {
     ...options,
     fontEmbedCss: '',
-    pixelRatio: 1,
+    pixelRatio: window.devicePixelRatio,
```

Comment:
This is the default value: https://github.com/bubkoo/html-to-image/blob/30000f088ef94878e2fa67dfcf89a09c13a0e14c/src/util.ts#L140

Reply:
ahh. right on. will update

Reply:
so it should be `pixelRatio: window.devicePixelRatio || 1`?

Reply:
Since that is the default, my understanding is that this line can be removed

Reply:
we can just omit passing in the `pixelRatio` arg completely cus the package defaults to the behavior we want